deps/v8/include/libplatform/v8-tracing.h

```diff
@@ -59,11 +59,19 @@ class V8_PLATFORM_EXPORT TraceObject {
       const char** arg_names, const uint8_t* arg_types,
       const uint64_t* arg_values,
       std::unique_ptr<v8::ConvertableToTraceFormat>* arg_convertables,
+#if defined(V8_OS_ZOS)
+      unsigned int flags, int pid, pthread_t tid, int64_t ts, int64_t tts,
+#else
       unsigned int flags, int pid, int tid, int64_t ts, int64_t tts,
+#endif
```

Comment:
Should we change this to pthread_t for all platforms since pthread_t is effectively an int on other platforms.

Reply:
Hmm, there's a few other places below where int is used instead of pthread_t for the other platforms.    I suggest we revisit this for master.zos and leave as is for v14.x.zos.

Reply:
Issue tracked in https://github.ibm.com/open-z/node-build/issues/237